karmagrambot/analytics.py

```diff
@@ -37,4 +37,40 @@ def get_karma(user_id: int, chat_id: int) -> int:
             elif reply['vote'] == '-':
                 bad_karma += 1
 
-    return good_karma - bad_karma
\ No newline at end of file
+    return good_karma - bad_karma
+
+def get_top_10_karmas(chat_id: int) -> list:
+    """Get the top 10 karmas in a given group, if the doesn't have enough users, return the total amount.
+
+    Args:
+        chat_id: The id of the chat that we're interested in.
+
+    Returns:
+        A list with top 10 users with better karma.
+    """
+
+    db = dataset.connect(DB_URI)
+    users = db['tracked'].find(chat_id=chat_id)
+    users_id = [u['user_id'] for u in users]
+
+    user_karma = {user_id: get_karma(user_id, chat_id) for user_id in users_id}
+
+    sorted_users_id = sorted(user_karma, key=user_karma.get, reverse=True)
+
+    sorted_users = []
+    for i in range(len(sorted_users_id)):
+        user_id = sorted_users_id[i]
+        user = db['users'].find(user_id=user_id)
+
+        for u in user:
```

Comment:
Refactor this into a `user_name(user: Dict[str, str]) -> str` function and this `for` can be rewritten as

`user_name(*user)`

Because you are (should be) absolutely sure that `user` is of size 1.